### DISCUSSION TITLE
how can I use in my own client

### DISCUSSION THREAD

**Discussion by raymond123456** [MadcowD/ell#368] [Q&A]
as my company offer a client called weopenai just originated from openai, I want to use in this project. how can I apply this client

**Comment by lingster** [comment]
if you are using something like olama you could do this:
```
client = openai.Client(base_url="http://localhost:11434/v1")
@ell.complex(model="gpt-4o", temperature=0, max_tokens=4096, client=client_gpt)
def func():
  ...
```
